openwisp_radius/models.py

```diff
@@ -103,3 +106,13 @@ def _get_instance(self, **kwargs):
     class Meta(AbstractRadiusUserProfile.Meta):
         abstract = False
         swappable = swappable_setting('openwisp_radius', 'RadiusUserProfile')
+
+
+class OrganizationRadiusSettings(OrgMixin, models.Model):
+    id = models.UUIDField(default=uuid.uuid4, primary_key=True)
+    token = models.CharField(max_length=15)
+    active = models.BooleanField(default=True)
+
+    def save(self, *args, **kwargs):
+        cache.delete(str(self.id))
```

Comment:
you are using `instance.pk` in the rest of the code, you should do the same here for consistency.

Some points:

1. wouldn't it be better to do this after the save operation is done? eg:

```python
result = super(OrganizationRadiusSettings, self).save(*args, **kwargs)
# update cache
```
If for some reason the query fails (think about a database failure), we won't update the cache, which sound more correct to me.

2. instead of deleting, can we simply update the cache?

3. I think we need also to override the `delete` method and use `cache.delete` there